### ISSUE TITLE
"idea" alias for "gen-idea"

### ISSUE THREAD

**Issue by huntc** [mpeltonen/sbt-idea#231]
Would you consider creating an alias of "idea" for the gen-idea task? We have done something similar with Play so that we have the IDEA creation look similar to the Eclipse creation i.e. "sbt eclipse" generates an Eclipse project.

I believe that tasks should contain a verb, but the situation is what it is for us.

An alias of "idea" would retire some code within Play.

Thanks for the consideration.

**Comment by adrien-aubel** [comment]
+1

**Comment by shellyan** [comment]
:+1:

**Comment by justjoheinz** [comment]
Good idea, And while at it, support the eclipse plugin logic of "idea with-sources=true", please
